src/olcs/FeatureConverter.js

```diff
@@ -176,9 +176,26 @@ class FeatureConverter {
         })
       });
     } else {
-      primitive.appearance = new Cesium.PerInstanceColorAppearance(options);
+      primitive.appearance = new Cesium.MaterialAppearance({
+        options,
+        material: new Cesium.Material({
+          translucent: color.alpha !== 1,
+          fabric: {
+            type: 'Color',
+            uniforms: {
+              color,
+            }
+          }
+        })
+      });
+      if (feature.get('olcs_shadows') || layer.get('olcs_shadows')) {
+        primitive.shadows = 1;
+        if (!this.scene.shadowMap.enabled) {
+          this.scene.shadowMap.enabled = true;
+          this.scene.globe.enableLighting = true;
+        }
```

Comment:
I understand that shadows will not work without this block.
But I don't think it is correct to enable these scene options automatically behind the scene.

You should:
- document `olcs_shadows` somewhere and explain there that you need these options enabled;
- you should enable them in the example and add a  comment there too.